python/create_mask.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+import glob
+import sys
+import mahotas
+import numpy as np
+import matplotlib.pyplot as plt
+
+if len(sys.argv) > 1:
+    path = sys.argv[1]
+else:
+    path=''
+
+f=glob.glob(path+"/meta/*ch_location*.txt")
+
+file=open(f[0],"r")
```

Comment:
Use a context manager to manipulate files, that way you won't forget to close it.

```python
with open(f[0], "r") as file:
   ...read the file
```